ziti-controller/subcmd/run.go

```diff
@@ -57,8 +57,9 @@ func run(cmd *cobra.Command, args []string) {
 		}
 
 		ec.SetHostController(c)
+		ec.Initialize()
 
-		go ec.InitAndRun()
+		go ec.RunAndWait()
 
 		if err = c.Run(); err != nil {
```

Comment:
RunAndWait calls Run then will 'wait for shutdown'...  Run is then called again here in this if.  Is that really 'correct'? seems strange to me

Reply:
oh, i see that one's calling run on "ec" and one's on "c" so i expect this is correct.

Reply:
You are correct.